src/system/data-transfer-tax-rates.md

```diff
@@ -46,7 +49,7 @@ If you conduct business in several states and ship a large quantity of product,
 
 ## Step 2: Prepare the Import Data
 
-You now have two spreadsheets open, side by side. One with the Magento export file structure, and the other with the new tax rate data that you want to import.
+You have two spreadsheets open, side by side. One with the Magento export file structure and the other with the new tax rate data that you want to import.
```

Comment:
The second sentence is one long fragment. 
```suggestion
You have two spreadsheets open, side by side. One contains the Magento export file structure, and the other contains the new tax rate data that you want to import.
```

Reply:
thanks for the reword